server/routerlicious/packages/protocol-base/src/quorum.ts

```diff
@@ -274,12 +314,16 @@ export class Quorum extends TypedEventEmitter<IQuorumEvents> implements IQuorum
                 });
             }
             if (value <= this.minimumSequenceNumber) {
-                return false;
+                return undefined;
             }
         }
 
+        const result: IQuorumUpdateMsnResult = {
```

Comment:
I believe this code always never executes, because caller updated this.minimumSequenceNumber to be equal to value (and the check above).